Lesson27-Loops&Conditions/scripts/main.js

```diff
@@ -0,0 +1,301 @@
+        console.log('All the odd numbers in your list is: ' + oddNumbers);
+        console.log('The length of your odd number list is: ' + oddNumbers.length);
+    }
+    // numberStats()
+
+
+// 6. Loop a calculator. Request 2 numbers and a sign, solve the problem, display the result and ask if the user wants one more.
+//      The loop continues until the user refuses. 
+//[[BUGG: MULTIPLY DOESN'T MULTIPLY]]
+function loopCalculator() {
+    do {
+        let numberX = parseInt(prompt('Please enter a number x:'))
+        let numberY = parseInt(prompt('Please enter a number y:'))
+        let sign = prompt('Please choose a sign: +, -, /, * ')
+    
+        if (sign == '+') {
+            let addSum = numberX + numberY;
+            console.log('The calculation of x and y is: ' + addSum);
+            question = prompt('Would you like to play again?');
+            console.log(question);
+            continue;            
+        }
+        if (sign == '-'); {
+            let minusSum = numberX - numberY;
+            console.log('The calculation of x and y is: ' + minusSum);
+            question = prompt('Would you like to play again?');
+            console.log(question);
+            continue;
+        }
+        if (sign == '/'); {
+            let devideSum = numberX / numberY;
+            console.log('The calculation of x and y is: ' + devideSum);
+            question = prompt('Would you like to play again?');
+            continue;
+        }
+        if (sign == '*'); {
+            let multiplySum = (numberX * numberY);
+            console.log('The calculation of x and y is: ' + multiplySum);
+            question = prompt('Would you like to play again?');
+            continue;
+        }
+    } while (question !== null);
+}
+// loopCalculator()
+
+
+// 7. Request a number from a user and ask by how many digits to move it. Move the digits and display the result
+// (if the number 123456 needs to be moved by 2 digits, the result will be 3456 12).
+function moveNumber () {
+    // Request number and convert it to an array
+    let userInput = prompt('Enter numbers, sperated with a comma:');
+    let userInputSplit = userInput.split(',');
+    userInputArray = userInputSplit.map(Number);  
+    // Request by how many digits to move array
+    let moveDigit = parseInt(prompt('By how many digits do you want to move the number?'))
+    // Slice array at moving point + create two arrays
+    let array1 = userInputArray.slice(0, moveDigit);
+    var array2 = userInputArray.slice(moveDigit, moveDigit + userInputArray.length);
+    // Join two arrays
+    let newArray = array2.concat(array1)
+    console.log('Your new list of numbers is: ' + newArray)
+}
+// moveNumber()
+
+
+// 8. Loop day input like this: «Day of the week. Would you like to see the next one?» which continues as long as the user clicks OK.
+// [[BUG: LOOP CAN'T GO BACK TO MONDAY. STOP'S AT SUNDAY]]
```

Comment:
But there is no loops in this function.

Reply:
We've done this in class - I'll update my code. Thank you.